Source/Scene/Expression.js

```diff
@@ -198,6 +242,26 @@ define([
             }
             val = createRuntimeAst(expression, args[0]);
             return new Node(ExpressionNodeType.UNARY, call, val);
+        } else if (call === 'RegExp') {
+            if (args.length === 0) {
+                return new Node(ExpressionNodeType.LITERAL_REGEX, new RegExp());
+            }
+            val = args[0];
+            if (args.length > 1) {
+                try {
```

Comment:
What could throw here?  Just the `RegExp` constructor?  If so, separate that out into its own block.